### DISCUSSION TITLE
500 错误 频繁出现

### DISCUSSION THREAD

**Discussion by thylaco1eo** [BurnHair/docs#177] [技术支持]
管理员您好，
我最近两周在NextChat客户端使用GPT4-turbo-preview模型的时候不定时出现500错误。账户登录正常。网页端聊天使用正常。
看到另一个帖子您说500错误是偶发现象，我就没管。但是最近出现的越来越频繁。请您查一下我这边的连接是不是有什么问题。是不是和我用的VPN有关？

{
  "error": {
    "message": " (request id: 20240801031424912853943EnYXuOT0)",
    "type": "upstream_error",
    "param": "500",
    "code": "bad_response_status_code"
  }
}

**Comment by BennyThink** [comment]
可否提供请求体，方便我排查问题？另外 `gpt4-turbo-preview` 似乎已经过时了

**Reply by thylaco1eo** [reply]
不好意思，最近比较忙，还没怎么有时间去处理这个事情。我先暂时把这个问题关闭，等有时间抓一下请求体再来找您吧。辛苦您
